dev-util/plan9port/plan9port-99999999.ebuild

```diff
@@ -0,0 +1,126 @@
+# Copyright 1999-2020 Gentoo Authors
```

Comment:
Why the filename is `99999999` instead of standard `9999`?

Reply:
The version of the other ebuild is a date with 8 characters (20200330).

Reply:
> I personally use it. I think it makes sense to have one because upstream plan9port doesn't actually have set releases.

Okay, thanks for clarifying. Because this was already in tree, I thought this is how they name their releases. There is a common way to write ebuilds for programs that have no releases. You name them `program-0_pre20200330`, so when 1.0 or even 0.1 release is made, it will always be an upgrade. Seeing this, I now strongly believe this should be re-versioned such. It will cause a downgrade after this PR is merged, but after that, it works like it's supposed to. And can co-exist with -9999 ebuild.